New Blog: Announcing the launch of the All In for Maintainers DEI Resource Hub (#89)
yuhattor/hattori.dev: content/githubblog/2023-02-01-announcing-the-launch-of-the-all-in-for-maintainers-dei-resource-hub.md

---

> title: "All In for Maintainers DEI リソースハブを開設しました。"

Reviewer: ```suggestion
title: "Maintainers DEI Resource のための All In を開設しました。"
```

englishsubtitle: "The DEI Resource Hub is a vetted collection of resources, tools, and best practices designed to help open source maintainers create and maintain inclusive and diverse open source communities."
englishtitle: "Announcing the launch of the All In for Maintainers DEI Resource Hub"
date: "2023-02-01"
cardurl: "https://github.blog/2023-02-01-announcing-the-launch-of-the-all-in-for-maintainers-dei-resource-hub/"
author: "Sarah Oyetubo"
description: "In 2021, we launched All In to provide the open source community with an opportunity to work together to create a powerful movement for diversity, equity, and inclusion (DEI). From supporting over 300 students through our All in for Students program, to capturing 7,000+ voices in the 2021 Open Source DEI Survey and hosting 300 community members in the Maintainers Listening Tour , we have listened, we have learned, and now we are ready to build the next initiative for All In."
coverimage: "https://github.blog/wp-content/uploads/2022/05/Community-Open-Source@2x.png?resize=1600%2C850"
category: "Community,Open Source,diversity,inclusion,social impact"
---

> <p>2021年、私たちは<a href="https://allinopensource.org/"> All Inを</a>立ち上げ、オープンソースコミュニティに、多様性、公平性、包括性（DEI）のための強力なムーブメントを生み出すために協力する機会を提供しました。<a href="https://allinopensource.org/access/">All in for Students</a>プログラムによる 300 人以上の学生の支援から、2021<a href="https://www.linuxfoundation.org/research/the-2021-linux-foundation-report-on-diversity-equity-and-inclusion-in-open-source">Open Source DEI Survey</a>における 7,000 人以上の声の収集、<a href="https://allinopensource.org/community/">Maintainers Listening Tour</a> における 300 人のコミュニティメンバーの受け入れまで、我々は耳を傾け、学び、そして今 All In の次のイニシアティブを構築する準備が整っています。</p>

Reviewer: ```suggestion
<p>2021年、私たちは<a href="https://allinopensource.org/"> All In </a>を立ち上げ、オープンソースコミュニティに、多様性、公平性、包括性（DEI）のための強力なムーブメントを生み出すために協力する機会を提供しました。<a href="https://allinopensource.org/access/">All in for Students</a>プログラムによる 300 人以上の学生の支援から、2021<a href="https://www.linuxfoundation.org/research/the-2021-linux-foundation-report-on-diversity-equity-and-inclusion-in-open-source">Open Source DEI Survey</a>における 7,000 人以上の声の収集、<a href="https://allinopensource.org/community/">Maintainers Listening Tour</a> における 300 人のコミュニティメンバーの受け入れまで、我々は耳を傾け、学び、そして今 All In の次のイニシアティブを構築する準備が整っています。</p>
```

<h2 id="supporting-maintainers-in-building-more-inclusive-communities">より包括的なコミュニティを構築するために、メンテナを支援する<a href="#supporting-maintainers-in-building-more-inclusive-communities" class="heading-link pl-2 text-italic text-bold" aria-label="Supporting maintainers in building more inclusive communities"></a></h2>
<p>メンテナーは、オープンソースコミュニティにとって、組織内のチームにとってのマネージャーのようなものです。メンテナーとコミュニティのリーダーは、コミュニティで起こることに直接的な影響力を持ちます。彼らは、オープンソースプロジェクトにおいて、誰かが歓迎されていると感じるか、歓迎されていないと感じるか、そのトーンを設定します。</p>
<p>このことを念頭に置き、私たちは、メンテナーの経験に焦点を当てることで、<a href="https://github.blog/2020-10-20-a-journey-of-diversity-inclusion-and-belonging-at-github/">オープンソースの多様</a>性とインクルージョンへの取り組みに焦点を当てました。2021年、私たちは「<a href="https://allinopensource.org/community/">メンテナリスニングツアー」を</a>実施し、コミュニティ内で多様性、公平性、インクルージョンを推進する際にメンテナが直面する重要な課題を明らかにするのに役立てました。</p>
<p class="purple-text text-gradient-purple-coral mt-6 mb-6">インクルージョンはコミュニティレベルで行われます。</p>

> <p>コミュニティやプロジェクトでDEIを推進する際に、何から手をつければいいのか、誰にサポートを求めればいいのかわからないというのは、メンテナンス担当者からよく聞く課題の一つです。そのため、DEIのスタート地点となるリソースを一箇所で見つけることができるウェブサイトやレポジトリを望んでいることがわかりました。</p>

Reviewer: ```suggestion
<p>コミュニティやプロジェクトでDEIを推進する際に、何から手をつければいいのか、誰にサポートを求めればいいのかわからないというのは、メンテナンス担当者からよく聞く課題の一つです。そのため、DEIのスタート地点となるリソースを一箇所で見つけることができるウェブサイトやリポジトリを望んでいることがわかりました。</p>
```

> <p>そこで、私たちはこのサイトを作りました。私たちは、オープンソースの実践者とDEIの思想的リーダーのコミュニティとして、一緒に作りました。以下は、このハブで見つけることができるコンテンツの種類の例です。</p>

Reviewer: ```suggestion
<p>そこで、私たちはオープンソースの実践者とDEIの思想的リーダーのコミュニティとして、このサイトを一緒に作りました。以下はこのハブで見つけることができるコンテンツの種類の例です。</p>
```

<ul>

> <li><strong>包括的なリーダーシップ。</strong>

Reviewer: ```suggestion
<li><strong>包括的なリーダーシップ</strong>
```

<ul>

> <li><a href="https://opensource.com/article/22/6/attract-contributors-open-source-project">オープンソースプロジェクトに信頼性をもって貢献者を引き寄せる。</a></li>

Reviewer: ```suggestion
<li><a href="https://opensource.com/article/22/6/attract-contributors-open-source-project">オープンソースプロジェクトに信頼性をもってコントリビューターを引き寄せる。</a></li>
```

<li><a href="https://github.com/mozilla/inclusion/blob/master/evaluation_tools/governance-basic.md">プロジェクトのためのオープンソース・インクルージョン基本チェックリスト</a> </li>
</ul>
</li>
<li><strong>アクセシビリティ</strong>
<ul>
<li><a href="https://opensource.com/article/21/7/open-source-neurodiversity">ADHD、失読症、自閉スペクトラム症の人のためのオープンソースにおけるアクセシビリティ</a></li>
<li><a href="https://www.w3.org/WAI/perspective-videos/">視覚障害者のためのLinuxのリミックス</a></li>
</ul>
</li>
<li><strong>バーンアウトとウェルビーイング</strong>
<ul>
<li><a href="https://www.infoworld.com/article/3563326/what-does-an-open-source-maintainer-do-after-burnout.html">オープンソースのメンテナは燃え尽きた後どうするのか？</a></li>
<li><a href="https://chaoss.community/kb/metric-project-burnout/">プロジェクトの幸福</a></li>
</ul>
</li>
</ul>
<h2 id="contributing-to-open-sourcing-diversity-equity-and-inclusion">オープンソースの多様性、公平性、包括性への貢献<a href="#contributing-to-open-sourcing-diversity-equity-and-inclusion" class="heading-link pl-2 text-italic text-bold" aria-label="Contributing to open sourcing diversity, equity, and inclusion"></a></h2>

> <p>この共創とリソース共有のプロセスにおいて、私たちはこのハブを確実に更新し、包括的なコミュニティを作り、サポートするためにメンテナが必要とするものと一致させるつもりです。オープンソースコミュニティとソフトウェア開発者の人口統計が進化するにつれて必要となる会話や研究、ツール、データを提供し続けたいと考えています。そこで、あなたの出番ですあなたやあなたのコミュニティにとって有益で、共有したいDEIコンテンツがありますか？DEI Resource Hubに掲載するために、<a href="https://allinopensource.org/maintainers/share-your-resources/">DEI Content Submission formに</a>記入して、リソースを提出してください。</p>

Reviewer: ```suggestion
<p>この共創とリソース共有のプロセスにおいて、私たちはこのハブを確実に更新し、包括的なコミュニティを作り、サポートするためにメンテナが必要とするものと一致させるつもりです。オープンソースコミュニティとソフトウェア開発者の人口統計が進化するにつれて必要となる会話や研究、ツール、データを提供し続けたいと考えています。そこで、あなたの出番ですあなたやあなたのコミュニティにとって有益で、共有したいDEIコンテンツがありますか？DEI Resource Hubに掲載するために、<a href="https://allinopensource.org/maintainers/share-your-resources/">DEI Content Submission form</a>に記入して、リソースを提出してください。</p>
```
